schema/data/prod/ciip_application_wizard.sql

```diff
@@ -1,15 +1,15 @@
 begin;
 
```

Comment:
Since we're not truncating anymore, should we make sure to set all the existing forms to inactive before upserting the prod ones?

```suggestion

update ggircs_portal.ciip_application_wizard set is_active=false;

```